frontend/src/component/project/Project/ProjectStatus/ProjectActivity.tsx

```diff
@@ -0,0 +1,90 @@
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import { useProjectStatus } from 'hooks/api/getters/useProjectStatus/useProjectStatus';
+import ActivityCalendar, { type ThemeInput } from 'react-activity-calendar';
+import type { ProjectActivitySchema } from '../../../../openapi';
+import { styled, Tooltip } from '@mui/material';
+
+const StyledContainer = styled('div')(({ theme }) => ({
+    gap: theme.spacing(1),
+}));
+
+type Output = { date: string; count: number; level: number };
+
+export function transformData(inputData: ProjectActivitySchema): Output[] {
+    const resultMap: Record<string, number> = {};
+
+    // Step 1: Count the occurrences of each date
+    inputData.forEach((item) => {
+        const formattedDate = new Date(item.date).toISOString().split('T')[0];
```

Comment:
Aren't they already formatted this way?

But anyway, this is just an accumulation, right? Assuming we have multiple entries for each date, it'll count each entry as one and sum them together? 

Maybe I've misunderstood the API, but I thought we returned each date only once, with already summed up activities? In that case, this'd be wrong, right?